index.html

```diff
@@ -11,13 +11,14 @@
 
 <body>
 
-    <section class="pop-up" style="display: none;">
-        <header class="pop-head">
+   <!-- The pop-up section -->
+	   <div class="pop-up">
```

Comment:
You have done a great job styling and creating your mobile menu so far. However, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are required to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) in fonts, colors, layouts etc. Currently, you have a few styling errors which can be made to make your project better:

- [ ] I have realized that there is a required background image that should appear in the mobile menu overlay which is currently not in your project. Kindly make sure you add them so that your project can meet all the project requirements.

**This is what your project currently looks like this:**

![image](https://user-images.githubusercontent.com/86318284/179817306-bd7423b5-3b7a-4930-93cc-cb1e6eca26f7.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/179817442-f9658105-1f95-4d6c-8868-6428f29bba42.png)

- [ ] Kindly make your hamburger menu smaller so that it matches the one in the Figma design.

**This is how your hamburger menu looks like:**

![image](https://user-images.githubusercontent.com/86318284/179817678-8af1ec8a-dad1-4e78-b77c-160e59c2180a.png)

**This is the desired Figma design hamburger menu size**

![image](https://user-images.githubusercontent.com/86318284/179817869-e8e4584b-a57d-44a4-9141-4b3936bd83da.png)

Reply:
DONE!

Reply:
- [ ] Like the previous reviewer requested, you need to make sure the background images are displaying correctly. Please fix the URLs in `background-image` 
In order to fix you need to add `""` around the URLS and remove the `/MYPORTFOLIO` from your URLS
### Do the following to all the background image URLS in your project:
**For example:**
In your `main.css` file  `background-image: url(/MYPORTFOLIO/images/pop-background1.png);` should be `background-image: url("/images/pop-background1.png");`